csrc/python_frontend/translation_utils.cpp

```diff
@@ -0,0 +1,107 @@
+
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+//
+#include <python_frontend/translation_utils.h>
+
+namespace nvfuser::python_frontend {
+
+std::string getString(const BinaryOp* bop) {
```

Comment:
`binary_op_type2string` doesn't always return the same string as `getString`, which corresponds with the names in `python_bindings.cpp`.
e.g., `BinaryOpType::Eq` return "equals" for `binary_op_type2string` vs "eq" in `getString`.

Reply:
Ha, that's subtle. Thanks for pointing that out. 🙇 

I'm wondering if `binary_op_type2string` is used anywhere in codegen? I guess not since there are things like `greaterThanOrEqual`...
Does it make sense to update `binary_op_type2string` to match `python_binding.cpp`'s string, which is customer facing already.

cc'ing @naoyam

Reply:
BTW, I'm not blocking merging the PR on this one. We can resolve it later if we decide to merge the two.